pkgs/development/libraries/shine/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, stdenv, fetchFromGitHub, autoreconfHook }:
+
+stdenv.mkDerivation rec {
+  pname = "shine";
+  version = "3.1.1";
+
+  src = fetchFromGitHub {
+    owner = "toots";
+    repo = "shine";
+    rev = version;
+    sha256 = "06nwylqqji0i1isdprm2m5qsdj4qiywcgnp69c5b55pnw43f07qg";
+  };
+
+  nativeBuildInputs = [ autoreconfHook ];
+
```

Comment:
If you get the sources from the released tarball at <https://github.com/toots/shine/releases/download/3.1.1/shine-3.1.1.tar.gz>, you won’t need the `autoreconfHook`.

Reply:
Is that really the preference in NixOS? I figured we'd always want to build from the "actual source" and not the processed sources

Reply:
I don’t know.